src/templates/landing_pages/introducing_usb.jsx

```diff
@@ -0,0 +1,356 @@
+import React from 'react';
+import Layout from './_common/layout.jsx';
+
+const UsbPage = () => (
+    <Layout
+        meta_description={it.L('[_1] Introducing USB', it.broker_name)}
```

Comment:
do we need translations for this? if not it can just be ```{`${it.broker_name} Introducing USB`}```

Reply:
I am not sure if we do

Reply:
as per this line https://github.com/binary-com/binary-static/pull/4517/files#diff-77228b84b3d4618d6fa968aeff50f04cR117 we are not showing this page in any language other than English, and you're not translating any other strings.

Reply:
noted